### PR TITLE
Adding VeraCrypt

### PR DESCRIPTION
**Pull request by frantzmiccoli**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/veracrypt.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'veracrypt' do
+  version '1.17'
+  sha256 'fa62b0e84f993dc2c58966e4eef91d8fdd3710913f78d62aa352e12187259f89'
+
+  url 'https://download-codeplex.sec.s-msft.com/Download/Release?ProjectName=veracrypt&DownloadId=1537180&FileTime=130999181375130000&Build=21031'
```
Comment (vitorgalvao) [inline Casks/veracrypt.rb]: Needs a `# https://download-codeplex.sec.s-msft.com/Download/Release?ProjectName=veracrypt was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (alebcay) [inline Casks/veracrypt.rb]: Is this a permalink? If so, I'd like to see if we can add back [some previous CodePlex links that kept breaking](https://github.com/caskroom/homebrew-cask/issues/6618).

Comment (vitorgalvao) [inline Casks/veracrypt.rb]: @alebcay This is the actual link they have on the page (i.e. it isn’t made on the fly just after you try to download), so it should keep working.

Comment (frantzmiccoli) [inline Casks/veracrypt.rb]: @vitorgalvao I guess that your point to @alebcay  is also answering your question. The project is hosted on CodePlex (which is offered by Microsoft), the project home page links to this CodePlex url which is somewhere on a Microsoft server. We have all reasons to believe that this is legit.

Comment (vitorgalvao) [inline Casks/veracrypt.rb]: @frantzmiccoli. I didn’t have a question, I made a request. That line is mandatory. See the links I’ve pointed you to and you will learn why.

You need to make the change.

**Comment by frantzmiccoli** [comment]
@vitorgalvao Done

File: Casks/veracrypt.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+cask 'veracrypt' do
+  version '1.17'
+  sha256 'fa62b0e84f993dc2c58966e4eef91d8fdd3710913f78d62aa352e12187259f89'
+
+  # https://download-codeplex.sec.s-msft.com/Download/Release?ProjectName=veracrypt&DownloadId=1537180&FileTime=130999181375130000&Build=21031 was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/veracrypt.rb]: Again, too long.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 38841c30824853ccf23d471158d5b0eac433fe86. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**